### DISCUSSION TITLE
Replicate Table vis features in Pivot Table v2

### DISCUSSION THREAD

**Discussion by LanceRadioactive** [apache/superset#19173] [Ideas]
**Is your feature request related to a problem? Please describe.**
Pivot Table v2 appears to be unfortunately unoptimized for viewing large swathes of data. In our experience with viewing monthly breakdowns (resulting in ~30k series) Superset and the underlying database can aggregate and provide a dataset reasonably quickly; but the resulting table forces the browser to hang as soon as it is prompted to render. This does not appear to be a problem for similar pivot tables of lesser size (for instance, 10k series).
This is remarkably not a problem for the Table vis even when it is prompted to do the exact equivalent of the pivot query simply due to its innate pagination, limiting the table to only show a number of rows at a time. Additionally, it supports individual formatting rules for each column, which is impossible in Pivot Table v2.

**Describe the solution you'd like**
- Replicate the pagination mechanics of the Table vis in Pivot Table v2 - the likely best would be by series, though even the outermost column/row would be a good start.
- Replicate individual formatting rules of the Table vis for aggregate cells in the Pivot Table v2 - similarly for row and column values would also be appreciated

**Describe alternatives you've considered**
Alternatively, this might be a local issue, though the display problems persist across various client machines, including relatively powerful ones. Advice on how to prevent such issues without loss of display would be appreciated - who knows, maybe it's literally just Firefox rendering problems?

**Comment by alan345** [comment]
Any news on that? Pivot Table v2is unfortunately unoptimized. We need a pagination to avoid pulling too many rows
